zilsd.adoc

```diff
@@ -16,12 +16,17 @@ The Zilsd extension adds the following RV32-only instructions:
 
 [NOTE]
 ====
-Implementations may need to crack these instructions, and perform to memory operations in sequence. Therefore, implementations are not required to ensure atomicity when storing to memory. Still, writing to the individual registers relating to a 64-bit operand of a load must happen atomically to ensure fault handling is possible.
+As the access size is 64-bit, accesses are only considered naturally aligned for effective addresses that are a multiple of 8. In this case, these instruction are guaranteed to not raise an address-misaligned exception.
+====
+
+[NOTE]
+====
+Implementations may need to crack these instructions, and perform two memory operations in sequence. Therefore, implementations are not required to ensure atomicity when storing to memory. However, writing to both registers written by a 64-bit load must happen atomically to ensure fault handling is possible.
```

Comment:
Minor nitpick: "writing to both registers written by a 64-bit load" -> "updates to both registers written by a 64-bit load" or similar to avoid close word repetition?